HomePage.cs

```diff
@@ -0,0 +1,58 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace Task70
```

Comment:
In general, your code looks good, but I would recommend you to add BasePage abstract class to store WebDriver variable.